padle/padle.py

```diff
@@ -0,0 +1,485 @@
+import discord
+import datetime
+import random
+import asyncio
+
+from typing import List, Literal, Optional
+from redbot.core import Config, commands
+from math import ceil
+
+# from dbcog.monster_graph import MonsterGraph # used for creating the filter
+from dbcog.dbcog import DBCog
+from tsutils.tsubaki.custom_emoji import get_awakening_emoji, get_rarity_emoji, get_type_emoji, get_emoji
+from tsutils.tsubaki.links import MonsterImage, MonsterLink
+from tsutils.tsubaki.monster_header import MonsterHeader
+
+from tsutils.cogs.userpreferences import get_user_preference
+from tsutils.helper_functions import conditional_iterator
+from tsutils.user_interaction import get_user_confirmation
+from tsutils.emoji import NO_EMOJI, SendableEmoji, YES_EMOJI
+
+from discordmenu.embed.components import EmbedThumbnail, EmbedMain
+from discordmenu.embed.view import EmbedView
+from redbot.core.utils.menus import (
+    menu,
+    next_page,
+    prev_page,
+    start_adding_reactions
+)
+
+
+class PADle(commands.Cog):
+    """PADle"""
+
+    def __init__(self, bot, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.bot = bot
+        self.config = Config.get_conf(self, identifier=123456789)
+        self.config.register_user(guesses=[], start=False, done=False, score=[], editID="", channelID="", embedText=[])
+        self.config.register_global(padleToday=1, storedDay=0, numDays=0, subs=[], allScores=[], saveDailyScores=[])
+
+        self._daily_padle_loop = bot.loop.create_task(self.generatePadle())
+
+    async def get_dbcog(self) -> "DBCog":
+        dbcog = self.bot.get_cog("DBCog")
+        if dbcog is None:
+            raise ValueError("DBCog cog is not loaded")
+        await dbcog.wait_until_ready()
+        return dbcog
+
+    @commands.group()
+    async def padle(self, ctx):
+        """Commands pertaining to PADle"""
+
+    @padle.command()
+    async def help(self, ctx):
+        """Instructions for PADle"""
+        await ctx.send("- PADle is similar to Wordle, except for PAD cards.\n"
+                       "- You have infinite tries to guess the hidden PAD card (chosen from a list "
+                       "of more-well known monsters). Everyone is trying to guess the same PAD card. "
+                       "With each guess, you are given feedback as to how similar the two cards are, "
+                       "including comparing the awakenings, rarity, typings, attributes, and monster "
+                       "point sell value.\n- A new PADle is available every day. Use `^padle start` to begin!")
+
+    @padle.command(aliases=["subscribe", "togglesub"])
+    async def togglesubscribe(self, ctx):
```

Comment:
Let's have a 2nd argument to this which is whether you want subscribe (as opposed to unsubscribe), default is True

so `^padle subscribe 0` to unsubscribe

When people subscribe tell them they can later run `[p]padle subscribe 0` to unsubscribe

(toggles are just kinda unexpected ux so it's best to avoid them)